dclab/definitions/feat_const.py

```diff
@@ -126,6 +130,11 @@
     ["mask", "Binary mask labeling the event in the image"],
     # See FLUOR_TRACES for valid keys
     ["trace", "Dictionary of fluorescence traces"],
+    # QPI features computed from holographic data
```

Comment:
Perhaps these should be in a special list? Or only the float32 ones? I don't have a good enough understanding to know if that makes sense.

Reply:
@paulmueller, for the non-scalar data-type, is this only defined in the RTDCWriter area of the codebase? I though it would also be here.

Reply:
No, I believe it's only in the RTDCWriter class.

Reply:
Great thanks!